Userland/Libraries/LibGfx/ICCProfile.cpp

```diff
@@ -234,15 +234,20 @@ ErrorOr<XYZ> parse_pcs_illuminant(ICCHeader const& header)
     return xyz;
 }
 
+template<size_t N>
+bool all_bytes_are_zero(const u8 (&bytes)[N])
+{
+    for (u8 byte : bytes)
+        if (byte != 0)
+            return false;
```

Comment:
Please add curly braces to multi-line `for`

Reply:
Done. Did we request a clang-format feature for this?

Reply:
Not that I know of, would be nice to have!

Reply:
Filed https://github.com/llvm/llvm-project/issues/59864.